Core/HTTPSUpgradeStore.swift

```diff
@@ -124,13 +146,13 @@ public class HTTPSUpgradePersistence: HTTPSUpgradeStore {
         }
     }
     
-    public func shouldUpgradeDomain(_ domain: String) -> Bool {
-        var result = true
+    public func shouldExcludeDomain(_ domain: String) -> Bool {
```

Comment:
Switched language to `shouldExclude` as `shouldUpgrade` was misleading - just because we _don't_ exclude a domain it doesn't mean we _should_ upgrade it!